cpmonitor/views.py

```diff
@@ -51,3 +63,38 @@ def task(request, city_slug, task_slugs):
             "subtasks": subtasks,
         },
     )
+
+
+@login_required
+def markdown_uploader(request):
+    if request.method != "POST":
+        return HttpResponse(_("Invalid request!"))
+    if "markdown-image-upload" not in request.FILES:
+        return HttpResponse(_("Invalid request!"))
+
+    image = request.FILES["markdown-image-upload"]
+    image_types = ["image/png", "image/jpg", "image/jpeg", "image/pjpeg", "image/gif"]
```

Comment:
I suggest we also allow .svg files to support vector graphics unless there's a reason not to - you already have a size limit in place, so that shouldn't be an issue.
If there is an argument against svg, we might want to add a comment here so we can remember not to add it.

Reply:
Alright, there *is* a good reason not to allow SVGs: they can include javascript, leading to potential XSS vulnerabilities.
There are workarounds, but they require a bit of work, so for now let's see how far we can get without SVG support.